### PR TITLE
Support linux-arm64

### PR DESCRIPTION
**Pull request by Zeneixe**
Next: Compile aarch64 executable: https://github.com/avh4/elm-format#development-info

### CODE REVIEW

File: package/npm/index.js [hunk 1]
```diff
@@ -16,6 +13,7 @@ module.exports = binwrap({
     "darwin-x64": root + "-mac-x64.tgz",
     "darwin-arm64": root + "-mac-x64.tgz",
     "linux-x64": root + "-linux-x64.tgz",
+    "linux-arm64": root + "-linux-arm64.tgz",
```
Comment (emancu) [inline package/npm/index.js:16]: 🤔 Maybe I'm missing something, shouldn't this be `aarch64`?

**Review by emancu — APPROVED** [review]

File: package/npm/index.js (hunk 1 above)
Comment (Zeneixe) [reply package/npm/index.js:16]: no... the error was saying arm64

**Review by PChambino — APPROVED** [review]

File: package/npm/index.js (hunk 1 above)
Comment (Zeneixe) [reply package/npm/index.js:16]:
```
Error: No binaries are available for your platform: linux-arm64
    at install (/apps/app/node_modules/binwrap/install.js:18:11)
    at Object.install (/apps/app/node_modules/binwrap/index.js:14:14)
    at Object.<anonymous> (/apps/app/node_modules/binwrap/bin/binwrap-install:18:9)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:75:12)
    at internal/main/run_main_module.js:17:47
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

Comment (Zeneixe) [reply package/npm/index.js:16]: arm64 and aarch64 are the same architecture anyways

**Pull request merged**